emukit/models/bohamiann.py

```diff
@@ -16,6 +12,36 @@
     """)
 
 
+import torch
+import torch.nn as nn
+def get_default_network(input_dimensionality: int) -> torch.nn.Module:
```

Comment:
This is very nice indeed, as an example. It shows how something a bit elaborate, based on PyTorch, can be used in Emukit